kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -85,6 +85,12 @@ logHomeOnPV: false
 # Otherwise, it is ignored.
 logHome: /shared/logs/domain1
 
+# The in-pod location for data storage of default and custom file stores.
+# If not specified, the value is derived from the domainUID as /shared/data/<domainUID>
+# This parameter is required if 'domainHomeOnPV' is true.
+# Otherwise, it is ignored.
```

Comment:
If not specified, the value should be ignored.   This is necessary IMO for backward compatibility.
Also, the value should not be required.  
This is necessary to (a) ensure backward compatibility of old operator domain configuration, (b) provide an intuitive way for users to tell us to use the configured value in the config.xml instead of the override value, and most of all (c) prevent dataloss on upgrade as otherwise the restarted stores will not be able to find their original data.

Reply:
Thanks Tom.  Just to clarify, so now we want to change requirement to:

If dataHome is not set or  is "" means means 'don't override'.

Reply:
Yes.

Reply:
Resolved.